data/src/main/java/me/androidbox/data/local/dao/AgendaDao.kt

```diff
@@ -0,0 +1,21 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.remote.model.AgendaModel
+
+@Dao
+interface AgendaDao {
+    @Query("SELECT * FROM ${DatabaseConstant.AGENDA_TABLE}")
+    fun getAgenda(): Flow<List<AgendaModel>>
```

Comment:
You never need to retrieve the whole agenda in this app, only for a specific day